### DISCUSSION TITLE
Changes on Installation FileNames

### DISCUSSION THREAD

**Discussion by agorosti** [HCL-TECH-SOFTWARE/connections-automation#85] [Q&A]
Hi,

I am trying to follow the quick-start-guide to deploy Connections with Connections pack,  but when I started to download the files needed from FlexNet download portal,  I realized that there is a lot of them that either I was unable to find or they seem to have a differente name.

For example,  I can not find LO100079-IC7.0.0.0-Common-Fix.jar  ,   but there is a  HCL_ConnectionsCP_7.0.0.1.zip.   What I should do on such as cases ?   

On a similar way, the WAS files have different names,  and  there is no FP16 or FP18,  no 8.0.6.15-WS-IBMWASJAVA-Linux.zip, 8.0.5.17-WS-IBMWASJAVA-Linux.zip, InstalMgr1.6.2_LNX_X86_64_WAS_8.5.5.zip, agent.installer.linux.gtk.x86_64_1.8.9006.20190918_1303.zip or agent.installer.linux.gtk.x86_64_1.9.1003.20200730_2125.zip

Any suggestion there ?

Thks !!

**Comment by dejo1307** [comment]
Hello Andres,

Check out this example, is this what you are looking for? 

https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/f23b5d2e7dae579119c4dbc0f8eca1400cae13b6/environments/examples/cnx6/db2/connections#L67-L86

If something else is missing in FlexNet please let me know and I can ask around. I used basically just basic account on IBM central for downloading and installing all those files, and those are the things we use also internally for all the tests. The stuff on the link (courtesy of Christoph) maps one names with the others. 

Regarding iFix, I will update it with the next PR, but these are the files you need (and variables you would need) to match what is in FlexNet:

cnx_ifix_installer="updateInstaller_2102.zip"
ifix_apar=LO100079
ifix_file=HCL_Connections_70_Update.jar

Thanks,
Dejan

**Reply by agorosti** [reply]
Dejan,

Thks..   I found that while main files where on FlexNet,  most of the files that I was unable to find (fixes related to was, etc) are only avaiable on IBM Fix Central.    Also, I find that while you name files as WAS_ND_V8.5.5_1_OF_3.zip, for example,  on flexnet they are using the part number for file name  (ie CIK2IML.zip)

Actually, there is a still a couple of files I can not find neither on flexnet, neither on IBM fix central,  those files are:

InstalMgr1.6.2_LNX_X86_64_WAS_8.5.5.zip
agent.installer.linux.gtk.x86_64_1.8.9006.20190918_1303.zip
agent.installer.linux.gtk.x86_64_1.9.1003.20200730_2125.zip


I guess that is because the release of Installation Manager used on the scrips is too old and is not loger available.  Doing some google I was able to find only  agent.installer.linux.gtk.x86_1.9.1004.20201109_1718 on IBM Fix Central. Any suggestion about what to do here  ?

I can not find either IBMConnectionsDocs_2.0.1.zip on Flexnet  (there is now a file called HCL_Docs_v201.zip that I assume is the one I need) and I could not find either SDI_7.2_XLIN86_64_ML.tar   (but I already saw you covered this last one on this thread -> https://github.com/HCL-TECH-SOFTWARE/connections-automation/discussions/44).

Additionally,  I was not able to find any of the files related to cp:

hybridcloud_20191122-120706.zip
hybridcloud_20200420-105909.zip
hybridcloud_20200825-172059.zip
hybridcloud_20201118-181644.zip

I am interested only on going to the latest connection pack (7),  and on flexnet there is an HCL_ConnectionsCP_7.0.0.1.zip,   not sure how is this file related to the "hybrid" files above.

Thks for your help !!!

**Reply by dejo1307** [reply]
Hi Andres,

Regarding IIM, I guess you want this one:

agent.installer.linux.gtk.x86_64_1.9.1003.20200730_2125.zip

I downloaded this one from IBM Central in December when customers were complaining that the other one we also tested is too old and not even available anymore. Are you saying that this one is not in FlexNet? (I apologise, I don't have access there myself). Btw this was the latest in IIM world in that moment, and since December we are using that one exclusively. 

For hybridcloud - to put it bluntly, that's however our Component Pack zip file is called in FlexNet. That's a working name of each build we test constantly, and we basically just use a symlink which gets updated every time there is new build (hence hybridcloud_latest.zip pointing to some other file with some timestamp). So you just need Component Pack package from FlexNet for v7 with ifix (I guess it is called 7.0.0.1). And you can (and should) use that one directly, there is no need to do anything older and then upgrade. 

I'll try to make those examples more 'friendly' to what is in FlexNet but that's quite a pain to maintain without having direct access there. 

Thanks for your understanding,
Dejan

**Reply by agorosti** [reply]
Sure I understand,  Thks a lot for your help and efforts.

 Unfortunately the dependence on HLC's / Flexnet packages should not be easy to mantain.    If I may help there any way to have a clear documentation, let me know,  I can provide you with some screenshoots and listing of the contains showed on Flexnet when trying to download Connections 7.

Regarding the IM,  I only find on IBM FIX Central what they call "refresh packs" that seems to be upgrades from previos IMs.  So, I was able to find he latest one agent.installer.linux.gtk.x86_1.9.1004.20201109_1718,   but not way to find InstalMgr1.6.2_LNX_X86_64_WAS_8.5.5.zip.    May be that one is only available at IBM Parner Download, but I have not entitlements there (only on HCL), so I have not access to the file (that I do not find on flexnet either).    May I use only agent.installer.linux.gtk.x86_1.9.1004.20201109_1718,  (since seems to be latest release) and forgot about InstalMgr1.6.2_LNX_X86_64_WAS_8.5.5.zip, agent.installer.linux.gtk.x86_64_1.8.9006.20190918_1303.zip and agent.installer.linux.gtk.x86_64_1.9.1003.20200730_2125.zip ?

**Reply by jmosen** [reply]
Hi Andres,
i had exactly the same problems you described. It was a lot of work to find all of the stuff from IBM. Regarding the Agent Installer I used agent.installer.linux.gtk.x86_64_1.9.1003.20200730_2125.zip and I left out the other Agent Installers.
